src/components/headers_overlay/headers_overlay.ts

```diff
@@ -197,6 +197,9 @@ abstract class AbstractResizer extends Component<ResizerProps, SpreadsheetChildE
       return;
     }
     if (this.state.waitingForMove === true) {
+      if (!this.env.model.getters.isGridSelectionActive()) {
+        this._selectElement(index, false);
+      }
       this.startMovement(ev);
```

Comment:
@rrahir Are we really disabling `startMovement` as mentioned in commit description?

Although IMO, `startMovement` should not be disabled to allow drag and drop of cols/rows while using 'SelectionInput'. (Gsheet allows it too)

Reply:
Yes because it really is moving the columns/rows, not just the selection.
I agree it could be nice if it would move the selection (in the selection input), but this is a stable release, we can't reinvent the feature. It can be an improvement for master (but it's not a priority)